src/main/java/org/folio/converter/Field008SplitterFactory.java

```diff
@@ -0,0 +1,177 @@
+package org.folio.converter;
+
+import io.vertx.core.json.JsonObject;
+
+import java.util.Arrays;
+import java.util.EnumMap;
+
+
+public class Field008SplitterFactory {
+  private static EnumMap<ContentType, Field008SplitStrategy> map = new EnumMap(ContentType.class);
```

Comment:
Diamond operator is missed:
`EnumMap<ContentType, Field008SplitStrategy> map = new EnumMap<>(ContentType.class);`